.drone.jsonnet

```
@@ -0,0 +1,191 @@
+          "npm install",
+        ],
+      ),
+      step(
+        name = "lint",
+        image = "node:10",
+        commands = [
+          "npm run lint",
+        ],
+      ),
+    ],
+  ),
+  pipeline(
+    kind = "pipeline",
+    name = "unit tests (node:6)",
+    steps = [
+      step(
+        name = "build information",
+        image = "node:6",
+        commands = [
+          "node --version",
+          "npm --version",
+          "git --version",
+        ],
+      ),
+      step(
+        name = "install",
+        image = "node:6",
+        commands = [
+          "npm install",
+        ],
+      ),
+      step(
+        name = "unit-test",
+        image = "node:6",
+        depends_on = [
+          "install",
+        ],
+        commands = [
+          "npm run test",
+        ],
+        environment = {
+          MONGO_URI: "mongodb://mongo:27017/key-recovery-service-test",
+        },
+      ),
+    ],
+    trigger = {
+      branch: {
+        exclude: [
+          "prod/production",
+        ],
+      },
+    },
+    services = [
+      {
+        name: "mongo",
+        image: "mongo:3.4",
+      },
+    ]
+  ),
+  pipeline(
+    kind = "pipeline",
+    name = "unit tests (node:10)",
+    steps = [
+      step(
+        name = "build information",
+        image = "node:10",
+        commands = [
+          "node --version",
+          "npm --version",
+          "git --version",
+        ],
+      ),
+      step(
+        name = "install",
+        image = "node:10",
+        commands = [
+          "npm install",
+        ],
+      ),
+      step(
+        name = "unit-test",
+        image = "node:10",
+        depends_on = [
+          "install",
+        ],
+        commands = [
+          "npm run test",
+        ],
+        environment = {
+          MONGO_URI: "mongodb://mongo:27017/key-recovery-service-test",
+        },
+      ),
+      step(
+        name = "audit",
+        image = "node:10",
+        failure = true,
```

Does this mean fail on error or ignore error?

---

It is close to "ignore error" and do not stop the run or fail entire pipeline.